components/tech/RoadmapList.vue

```diff
@@ -272,12 +245,11 @@ ul li {
       rgba(255, 255, 255, 0.8) 0,
       rgba(255, 255, 255, 0.216) 100%
     ),
-    linear-gradient(326.58deg, #ffe177 -3.92%, #ff975c 112.32%);
+    linear-gradient(326.58deg, #ffe177 -3.92%, #ff975c 112.32%) !important;
```

Comment:
why is this `!important`

Reply:
To override the existing gradient, which is not getting overridden due to some unknown reason, maybe some cascading issue or specificity issue. Will figure out later